src-built-in/components/searchMenu/searchMenu.css

```diff
@@ -33,7 +29,7 @@ body {
     height: 21px;
     display: flex;
     font-weight: 600;
-    color: var(--primary-accent-color);
+    color: var(--blue1);
```

Comment:
@DanNicolai this looks to be going backward from what we talked about, where we would reference thematic elements (accent-color) rather than direct colors (blue).